data/human/navy missions intro.txt

```diff
@@ -0,0 +1,1477 @@
+			`	You nod. "When I learned that one of our officers had threatened Hadley, I wasn't about to let something like that slide."`
+			`	"And I thank you for that - there's no way I could bring that to my boss, and anyone else would just ignore me. But with you here, I think we have a real chance at getting Hadley and his family to safety."`
+			`	"So what's the plan?"`
+			`Henry procures a small holo-projector. "When the bombings happened, Lucas and I knew we had to come up with a contingency in case things got ugly. After considering our options, we decided to select this place." The projector highlights Hermes in the Capella System.`
+			`	"Hermes?" you say, with a sly grin. "That makes perfect sense. The planet's got orbital bombardment stations trained on the planet around the clock - if someone tried to tail you there, it's almost certain they'd be atomized by the relays."`
+			`	"That's exactly what we were thinking. And it's still close enough to Syndicate Space that it wouldn't be that far of a trip. Ideally, we wanted to settle in the Deep, but that's too far away."`
+			`The pair of you stand up. "I'll be sure to relay this to my superior officer, Henry," you reply, shaking his hand. "He's been trying to come up with a plan to get Hadley to safety, and this will be the break he needs."`
+			`	"I want to thank you once again, Lieutenant...?"`
+			`	"<last>. <first> <last>, if you want, Henry."`
+			`	"Thank you, <first>. Really, I can't thank you enough."`
+			`A thought enters your head. "Henry...I know this might be asking a lot, but if it's possible, I have a favor to ask. I'm sure you're well-aware of the bombings of Geminus and Martini at this point - if you get any leads, can I ask you to be my eyes and ears?"`
+			`"You got it, <last>," he replies without hesitation. "After all, you're bending over backwards to make sure one of my best friends is safe and sound. I'll let my other buddies know to keep an eye out for you - the more feelers we can put out, the better."`
+			`The pair of you soon exit your ship and head back to the auxiliary captains. "I won't let you down, Lieutenant," Henry says as he gets back in line. "Thanks for giving me this chance!"`
+			label end
+			`It takes a couple hours before all the mechanics have loaded onto the auxiliaries. With your errand complete, you board the <ship> and get ready to head back to Geminus.
+
+	on complete
+		payment 2500000
+		dialog
+			`As you break through the clouds over Geminus, you are treated to the awe-inspiring sight of the new shipyard unfold before you. "Welcome to the new Geminus, <last>," Danforth congratulates you. "This represents the first step of many on the road to victory. To celebrate, take these 2.5 million credits - I'm sure there's something in the shipyard you might want."`
+
+event "geminus rebuilt"
```

Comment:
So the entire shipyard/facilities are rebuilt in... days? a month or two?
I know the player can take their time with some missions, but the shortest time would be ... pretty amazing for rebuilding a whole Navy shipyard.  FW forces you to take what... 2x30 day breaks, plus the whole campaign and 90 days after, for Geminus to be rebuilt.